zstacklib/zstacklib/utils/sharedblock_lanfree.py

```
@@ -0,0 +1,321 @@
+    from zstacklib.utils import linux
+    from zstacklib.utils import shell
+
+    cmd = shell.ShellCmd(
+        "blockdev --getsize64 %s" % linux.shellquote(path))
+    cmd(is_exception=False)
+    if cmd.return_code != 0:
+        raise Exception(
+            "LVM_RANGE_DEVICE_CAPACITY_MISSING: cannot read capacity for "
+            "device[%s], error[%s]" % (path, cmd.stderr))
+    try:
+        capacity = long(cmd.stdout.strip())
```

Ruff (0.16.2)

[error] Undefined name `long`

(F821)